shared/plan/service.py

```diff
@@ -191,15 +225,21 @@ def start_trial_manually(self, current_owner: Owner, end_date: datetime) -> None
         Returns:
             No value
         """
-        if self.plan_name not in FREE_PLAN_REPRESENTATIONS:
+        # Start a new trial plan for free users currently not on trial
+        if self.plan_name in FREE_PLAN_REPRESENTATIONS:
+            self._start_trial_helper(current_owner, end_date, is_extension=False)
+        # Extend an existing trial plan for users currently on trial
+        elif self.plan_name in TRIAL_PLAN_REPRESENTATION:
+            self._start_trial_helper(current_owner, end_date, is_extension=True)
+        # Paying users cannot start a trial
+        else:
             raise ValidationError("Cannot trial from a paid plan")
 
-        self._start_trial_helper(current_owner, end_date)
-
     def cancel_trial(self) -> None:
+        """Cancels the ongoing trial for the organization."""
         if not self.is_org_trialing:
             raise ValidationError("Cannot cancel a trial that is not ongoing")
-        now = datetime.utcnow()
+        now = datetime.now()
```

Comment:
That's a really good point, but since the API is currently using this functionality, I want to keep it as is. I don't want to interfere with the internal functionality to avoid unexpected errors, but it's definitely something to keep in mind for when i touch this file again